src/EditLabelDialog/EditLabelDialog.stories.jsx

```diff
@@ -0,0 +1,67 @@
+// write a story
+
+import { Button } from "@mui/material";
+import EditLabelDialog from "./EditLabelDialog";
+import React from "react";
+import { action } from "@storybook/addon-actions";
+
+export default {
+  component: EditLabelDialog,
+  title: "Dialog/EditLabelDialog"
+};
+
+const Template = args => {
+  const [open, setOpen] = React.useState(false);
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
```

Comment:
Rather than trying to handle the state yourself (and hence get out of sync with the storybook controls) we can use the `useArgs` hook from storybook to handle this state.

https://storybook.js.org/docs/addons/addons-api#useargs

See: https://github.com/IPG-Automotive-UK/react-ui/blob/5cb92d7a5eb62220004d8ee849d244560c179339/src/VehicleSelect/VehicleSelect.stories.tsx#L19-L31 for an example usage in RUI.